src/OrderBook.test.tsx

```diff
@@ -1,5 +1,143 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { OrderBook, Props, Layout } from './OrderBook';
+import '@testing-library/jest-dom/extend-expect';
+
+const book = {
+  type: 'snapshot',
+  product_id: 'BTC-USD',
+  asks: [
+    ['1166441.89', '7.3843470214'],
+    ['1162261.90', '1.50646551300'],
+    ['1166661.89', '7.38476560214'],
+    ['11224661.90', '1.506768551300'],
+  ],
+  bids: [
+    ['1165461.89', '7.384334470214'],
+    ['11676661.90', '1.5065681300'],
+    ['11643461.89', '7.38470457214'],
+    ['116564761.90', '1.50651300'],
+  ],
+};
+const propsSample: Props = {
+  applyBackgroundColor: false,
+  askColor: [235, 64, 52],
+  bidColor: [0, 216, 101],
+  book,
+  fullOpacity: false,
+  interpolateColor: () => [235, 64, 52],
+  layout: Layout.Row,
+  listLength: 2,
+  showHeaders: true,
+  showSpread: true,
+  spread: 'Hello',
+  stylePrefix: 'test_OrderBook',
+};
+
 describe('<OrderBook />', () => {
-  it('should pass', () => {
-    expect(true).toBe(true);
+  it('OrderBook should be rendered', () => {
+    const { getByText } = render(<OrderBook {...propsSample} />);
+
+    expect(getByText('Ask')).toBeInTheDocument();
+  });
+
+  it('Orderbook should display Spread when showSpread is true', () => {
+    const renderedOutput = render(<OrderBook {...propsSample} />);
+
+    expect(renderedOutput.queryByText('Spread')).toBeInTheDocument();
+  });
+
+  it('Orderbook should display Headers when showHeaders is true', () => {
+    const renderedOutput = render(<OrderBook {...propsSample} />);
+
+    expect(renderedOutput.queryByText('Ask')).toBeInTheDocument();
+    expect(renderedOutput.queryByText('Bid')).toBeInTheDocument();
+  });
+
+  const propsWithNoHeader = { ...propsSample };
```

Comment:
It would be nice to wrap _all_ of the related tests that rely on some variable in a `describe` block.

Also, you can make the definition of this variable a little more terse by combining the spread operator with the new key you want to define, like this:

```ts
const propsWithNoHeader = {
  ...propsSample,
  showHeaders: false,
};
```